aiohttp/client_exceptions.py

```diff
@@ -238,17 +239,34 @@ class InvalidURL(ClientError, ValueError):
 
     # Derive from ValueError for backward compatibility
 
-    def __init__(self, url: Any) -> None:
+    def __init__(self, url: StrOrURL, description: Optional[str] = None) -> None:
```

Comment:
Pls use `Union` for clarity.
```suggestion
    def __init__(self, url: StrOrURL, description: Union[str, None] = None) -> None:
```